### ISSUE TITLE
Can I use other node.js package with it? (Like websocket etc)

### ISSUE THREAD

**Issue by aeroxy** [mafintosh/node-on-android#16]
Can I use other node.js package with it, if so where should I store the /node_modules folder? Under /example/app?

**Comment by jguillod** [comment]
Did someone find the answer?
Just essential…

**Comment by aeroxy** [comment]
Yes you can. However if you are using compiled package you might not be able to compile due to the lack of compiler or permission.

**Comment by jguillod** [comment]
Thanks for the "Yes"... 

Can you also explain how to do it : where to install node_modules, set package dependencies, aso?

An example would be just Welcome !

**Comment by aeroxy** [comment]
You can either use console to install it via npm (of which you might need to install on any other linux), or you can install it on another linux and pack the whole folder and send it to your Android.

**Comment by aeroxy** [comment]
Oh I realized I might be answering the wrong thread. I gave up on this solution btw. I install Node on Android directly.

**Comment by jguillod** [comment]
@aeroxy : What do you mean by 

> I install Node on Android **directly**

A long while that I succeed installing NodeJS with Termius. But what I need is to build a standalone application running NodeJS for final users. Hacking with Termius is fine for dev test and play, but not for releasing apps through stores.
So, if you have a solution, please share in details the how-to. Thanks!
